compile broken on freebsd

```
../deps/osr/src/cpl_conv.cpp:2464:21: error: variable has incomplete type 'VSIStatBufL' (aka 'stat64')
        VSIStatBufL sStatBuf;
                    ^
../deps/osr/src/cpl_vsi.h:159:16: note: forward declaration of 'stat64'
typedef struct VSI_STAT64_T VSIStatBufL;
               ^
../deps/unix-include/cpl_config.h:209:22: note: expanded from macro 'VSI_STAT64_T'
#define VSI_STAT64_T stat64
                     ^
1 error generated.
gmake: *** [Release/obj.target/osr/deps/osr/src/cpl_conv.o] Error 1
```

--- a/deps/bsd-include/cpl_config.h
+++ b/deps/bsd-include/cpl_config.h
@@ -148,10 +148,10 @@
 #define HOST_FILLORDER FILLORDER_LSB2MSB
 
 /* Define as const if the declaration of iconv() needs const. */
-#define ICONV_CONST 
+#define ICONV_CONST const
 
 /* For .cpp files, define as const if the declaration of iconv() needs const. */
-#define ICONV_CPP_CONST 
+#define ICONV_CPP_CONST const
 
 /* Define to the sub-directory in which libtool stores uninstalled libraries.
    */
@@ -203,10 +203,10 @@
 /* #undef VSI_NEED_LARGEFILE64_SOURCE */
 
 /* Define to name of 64bit stat function */
-#define VSI_STAT64 stat64
+#define VSI_STAT64 stat
 
 /* Define to name of 64bit stat structure */
-#define VSI_STAT64_T stat64
+#define VSI_STAT64_T stat
 
 /* Define to 1 if your processor stores words with the most significant byte
    first (like Motorola and SPARC, unlike Intel and VAX). */

--- a/deps/osr.gyp
+++ b/deps/osr.gyp
@@ -12,7 +12,13 @@
           'include_dirs': [
              './win-include'
           ]
-        }, {
+        }],
+        [ 'OS=="freebsd"', {
+          'include_dirs': [
+             './bsd-include'
+          ]
+        }],
+        [ 'OS!="freebsd" and OS!="win"', {
           'include_dirs': [
              './unix-include'
           ]

